### ISSUE TITLE
[Programmatic Access - Make code - New Proje>Settings>>Print Code]: alt attribute is not defined for the images.

### ISSUE THREAD

**Issue by BURRAshiva01** [microsoft/pxt-microbit#6089]
### User Experience:

Visually impaired people who depend on screen readers for navigation can’t interpret an image directly; they rely on alternative text to communicate the image's meaning to users. The end user will not be able to know whether the image is decorative or meaningful if it doesn't have alt text or role of none or presentation.

**Note:** User credentials should NOT be included in the bug.

### Repro Steps:

1. Launch application URL: https://makecode.microbit.org/beta#
2. Home page will open.
3. Navigate "New Project" option and select it.
4. Editor screen will open.
5. Navigate to "Settings" option appearing in header section and select it.
6. Navigate to "Print" option and select it.
7. Navigate image controls appearing under "Print Code" page 
8. Verify in DOM alt attribute is defined for image controls or not.

### Actual Result:
**Issue:** The alt attribute is not defined for the images in the 'Print' dialog.

### Expected Result:
The alt attribute should be defined for informative images.

**Ex:** alt="on start" or alt="forever" for image controls.

### MAS Reference:
[MAS 1.1.1 – Non-text Content](https://aka.ms/MAS1.1.1)

<img width="960" alt="Image" src="https://github.com/user-attachments/assets/00dcc7d6-6728-4090-bf04-891a90134604" />

**Comment by BURRAshiva01** [comment]
#Accessibility; #A11yMAS; #A11ySev2; #MAS1.1.1; #HCL; #Benchmark; #Win11-Chrome; #BM-HCL-MakeCode-Feb2025; #BUGWithoutSR; #AILimited;

**Comment by Yamini234u** [comment]
As checked issue on https://www.microsoft.com/en-us/makecode/teach. The issue has been fixed. Hence, closing the bug
